images/libvirt-kubevirt/qemu-kube

```diff
@@ -93,24 +93,8 @@ set -e
 # to adhere to the resource limitations of the container.
 exec sudo -C 10000 unshare --mount bash -s << END
 
-  function _term() {
-    rm -rf /etc/resolv.conf.\$\$
-    pkill -P \$! --signal SIG\$1
-  }
-
-  trap "_term TERM" TERM
-  trap "_term INT" INT
-  trap "_term HUP" HUP
-  trap "_term QUIT" QUIT
-  trap "_term TERM" EXIT
-  trap "_term TERM" ERR
-
-  # Make sure qemu can resolve kubernetes DNS entries
-  nsenter -m -t $CONTAINER_PID cat /etc/resolv.conf > /etc/resolv.conf.\$\$
-  mount --bind /etc/resolv.conf.\$\$ /etc/resolv.conf
-
   cgclassify -g ${CGROUPS}:$CGROUP_PATH --sticky \$\$
-  nsenter -t $CONTAINER_PID -n -p $CMD &
+  nsenter -t $CONTAINER_PID -p $CMD &
```

Comment:
Please ignore this patch - it is part of a separate pull request, and only visible because github fails todo dependancies between PRs :-(